march_rqt_gait_generator/src/march_rqt_gait_generator/gait_generator_view.py

```diff
@@ -36,6 +37,12 @@ def __init__(self):
 
         self.gait_type_combo_box.addItems(['walk_like', 'sit_like', 'stairs_like'])
 
+        self.initialize_shortcuts(self)
+
+    def initialize_shortcuts(self, parent):
+        self.ctrl_z = QShortcut(QKeySequence(self.tr('Ctrl+Z')), parent)
+        self.ctrl_shift_z = QShortcut(QKeySequence(self.tr('Ctrl+Shift+Z')), parent)
```

Comment:
I think it is better to name these shortcuts `undo_shortcut` and `redo_shortcut` then it is more clear elsewhere in the code what they do and if you want to change the key sequence you don't have to rename your variables.

Reply:
I thought the other way around - this way the view only knows that ctrl-z is a shortcut and not what it does. If you want to change what ctrl-z does, you don't have to change anything in the view. Its kind of the controller knowing about the view or the view knowing about the controller question, so I would prefer to keep it like this.